src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/theme-switcher.html

```diff
@@ -1,7 +1,5 @@
-<span class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle"
-      title="{{ _('light/dark') }}"
-      data-toggle="tooltip">
-  <a class="theme-switch" data-mode="light"><i class="fa-solid fa-sun"></i></a>
-  <a class="theme-switch" data-mode="dark"><i class="fa-regular fa-moon"></i></a>
-  <a class="theme-switch" data-mode="auto"><i class="fa-solid fa-circle-half-stroke"></i></a>
-</span>
+<button class="theme-switch-button btn btn-sm btn-outline-primary navbar-btn rounded-circle" title="{{ _('light/dark') }}" aria-label="{{ _('light/dark') }}" data-toggle="tooltip">
```

Comment:
Didn't this `span -> button` change get merged already? We should double check that we are doing this from the latest commit

Reply:
I started this PR before the merge of #1050. when I merged it back in the line has been analyzed as a change (with the linting)